physics/cu_c3_sh.F90

```diff
@@ -6,12 +6,12 @@ module cu_c3_sh
     use progsigma, only : progsigma_calc
 
     !real(kind=kind_phys), parameter:: c1_shal=0.0015! .0005
-    real(kind=kind_phys), parameter:: c1_shal=0. !0.005! .0005
     real(kind=kind_phys), parameter:: g  =9.81
     real(kind=kind_phys), parameter:: cp =1004.
     real(kind=kind_phys), parameter:: xlv=2.5e6
     real(kind=kind_phys), parameter:: r_v=461.
-    real(kind=kind_phys), parameter:: c0_shal=.001
+    real(kind=kind_phys)  :: c0_shal=.004
```

Comment:
Is there a reason why these aren't parameters anymore? Will they be changed within the code?